R/generateMappingfile.R

```diff
@@ -0,0 +1,163 @@
+#' Generate mapping from the template containing all the metadata
```

Comment:
I adjusted the code in this function to always return a list with the generated comments and template. Not sure this would affect write.reportProject, though. But feel free to enhance the code and create a PR for this,